src/__tests__/__snapshots__/downshift.aria.js.snap

```diff
@@ -0,0 +1,107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`basic snapshot 1`] = `
+<div
+  aria-expanded="false"
+  aria-haspopup="listbox"
+  aria-labelledby="downshift-0-label"
+  aria-owns="downshift-0-menu"
+  role="combobox"
+>
+  <label
+    data-testid="label"
+    for="downshift-0-input"
+    id="downshift-0-label"
+  >
+    label
+  </label>
+  <input
+    aria-autocomplete="list"
+    aria-controls="downshift-0-menu"
+    aria-expanded="false"
+    aria-labelledby="downshift-0-label"
+    autocomplete="off"
+    data-testid="input"
+    id="downshift-0-input"
+    role="listbox"
+    value="item"
+  />
+  <button
```

Comment:
This button is essentially redundant. The input does all the work for you here, and focus should remain in the input when a user is interacting with the Combobox. If I actioned this, you'd have to focus me back into the input so I can interact with the options correctly. Feel free to keep an icon or something that visually depicts it's dropdown-y-ness, but it's easier to just remove the button.

The `listbox` often shows on focus on the input, or pressing enter and/or down arrow when focused in the input.

Reply:
What about mouse-only users?

Reply:
Click event on the input? The input is a much better pointer target in general because of its size. 

You can split the interaction too if you desire:

- onClick will show the listbox 
- onFocus won’t
- onKeyPress `down` and `enters will show

Or whatever suits the full desired interaction. Sometimes it’s useful to not show the dropdown as you’re tabbing through a form for example, and only show it when I desire, similar to a native select element.

Reply:
Having a "clear" button is great though! 👍